x-pack/plugins/lists/server/services/exception_lists/exception_list_client.ts

```diff
@@ -702,13 +705,36 @@ export class ExceptionListClient {
   }: ImportExceptionListAndItemsOptions): Promise<ImportExceptionsResponseSchema> => {
     const { savedObjectsClient, user } = this;
 
-    return importExceptionsAsStream({
+    // validation of import and sorting of lists and items
+    const readStream = createExceptionsStreamFromNdjson(maxExceptionsImportSize);
+    const [parsedObjects] = await createPromiseFromStreams<PromiseFromStreams[]>([
       exceptionsToImport,
-      maxExceptionsImportSize,
+      ...readStream,
+    ]);
```

Comment:
I pulled this in from `importExceptionsAsStream`. We seem to have what we need in `parsedObjects` to determine the data being imported.